R/plumber.R

```diff
@@ -220,6 +220,8 @@ plumber <- R6Class(
 
       # Initialize
       private$serializer <- serializer_json()
+      # Default parsers to maintain legacy features
+      private$parsers <- c(parser_json(), parser_query())
```

Comment:
```suggestion
      private$parsers <- c(
        parser_json(), 
        parser_query(),
        parser_text(),
        parser_octet(),
        parser_multi()
      )
```

* Adding `text` and `octet` as they seem like fair expected behavior given the new capabilities
* Adding `multi` to parse multipart forms.